billboardd/daemon.h

```diff
@@ -282,7 +288,9 @@ class Daemon : public QObject {
         }
 
         void contextChanged() {
-            updateTimer.start();
+            if (tklock.isLocked()) {
```

Comment:
Don't check for `isLocked()` here - it will break the preview in the GUI application (the screen is never locked when the user uses the GUI application).